src/components/trick-list/DifficultyGroup.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import { Trick } from '@/lib/database/daos/trick';
+
+import {
+  Collapsible,
+  CollapsibleContent,
+  CollapsibleTrigger,
+} from '@/components/ui/collapsible'
+import TrickOverviewCard from './TrickOverviewCard.vue';
+
+interface Props {
+  level: string
+  tricks: Trick[]
+}
+
+const { level, tricks } = defineProps<Props>();
+const isOpen = ref(true)
+</script>
+
+
+<script lang="ts">
+export default {
+  data() {
+    return {
+      isSticky: false
+    };
+  },
+  mounted() {
+    window.addEventListener("scroll", this.handleScroll);
+  },
+  beforeDestroy() {
+    window.removeEventListener("scroll", this.handleScroll);
+  },
+  methods: {
+    handleScroll() {
+      const rect = this.$el.getBoundingClientRect();
+      this.isSticky = rect.top <= 64;
+    }
+  }
+};
+</script>
```

Comment:
Just wondering: Why are there two <script> tags and both have different formats on how methods are declared?